tests/tests/test-trace-filter.ts

```diff
@@ -190,4 +190,21 @@ describeDevMoonbeam("Trace filter - Contract creation ", (context) => {
 
     expect(response.result.length).to.equal(4);
   });
+
+  it("should handle pagination", async function () {
+    let response = await customWeb3Request(context.web3, "trace_filter", [
+      {
+        fromBlock: "0x03",
+        toBlock: "0x04",
+        count: 2,
```

Comment:
so this is the number of pages? Not super explicit

Reply:
I'm not too familiar with the RPC, I copied the test from #324